DFC.App.FindACourse/wwwroot/StaticRobots.txt

```diff
@@ -1,3 +1,2 @@
-﻿User-agent: *
-
-Disallow: /StaticRobots.txt
+﻿User-agent: PowerMapper
```

Comment:
@defc0nONE is this an intentional change for fac beta?

Reply:
Judging from a DfE Slack conversation, this was edited to allow for a Methods consults to allow for Accessibility testing - we should revert this if that's the case

Reply:
Actually talking to Joe. I think we can leave this as is. Doesn't cause any issues

Reply:
Sure, sounds good to me.